src/Resources/config/services.xml

```diff
@@ -3,8 +3,8 @@
            xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
     <imports>
         <import resource="services/controller.xml"/>
+        <import resource="services/menu.xml"/>
```

Comment:
I would still have it in the event subscriber namespace, hehe. I know menu is the legacy from Sylius plugins, but I do not agree with it :D

Reply:
Again, it's probably a matter of taste :dancer: even though the class itself indeed is technically an event subscriber/listener, I feel keeping it in the Menu namespace gives much better information about what _it's responsible for_ rather than _what it is_. Still - let's make it an even subscriber for consistency 👍

Reply:
I agree on the responsibility argument. From my experience it seems that people most of the times only do this with the menu and all other event subscribers end up in the `EventSubscriber/EventListener` namespace. I find that inconsistent, and that's why I fell back to just having _all_ subscribers in the same namespace